src/services/lansweeper/baseRequest.ts

```diff
@@ -1,8 +1,9 @@
 import { isEmpty } from "lodash-es";
 import { proxyFetch, adminGenericProxyFetch } from "@deskpro/app-sdk";
-import { GRAPHQL_URL, placeholders } from "@/constants";
+import { GRAPHQL_URL, placeholders, INTEGRATION_ID } from "@/constants";
 import { getQueryParams, getRequestBody, parseGlobalAccessToken } from "@/utils";
 import { LansweeperError } from "./LansweeperError";
+import manifest from "@/../manifest.json";
```

Comment:
Feels weird importing something outside the root but I guess it makes sense in this case.

Reply:
Unfortunately, there are no other options _(or I don't know them)_. But it works.

<img width="1144" alt="Screenshot 2024-10-07 at 13 41 52" src="https://github.com/user-attachments/assets/a6595a9d-378c-4186-a488-dea28d02fbb4">